twoaxistracking/shading.py

```diff
@@ -35,6 +62,9 @@ def shaded_fraction(solar_elevation, solar_azimuth,
     slope_tilt : float
         Tilt of slope relative to horizontal [degrees]. Used to determine
         horizon shading.
+    max_elevation_angle : float, optional
+        The maximum elevation angle for which shading may occur. The shaded
+        fraction is set to zero if the solar elevation is above this.
```

Comment:
I'm not sure about this parameter name.  Maybe `max_shading_elevation` to be parallel to the function that calculates it?

Reply:
Perhaps also worth mentioning that this is just a speed optimization and should not change results (assuming the passed value is appropriate)

Reply:
Good point. I have added the following comments:

    max_shading_elevation : float, optional
        The maximum elevation angle for which shading may occur. Specifying the
        max_shading_elevation skips the calculations for which the solar
        elevation angles is higher and sets the shaded fraction to zero.
        This reduces the calculation time and results in the same output
        assuming the correct value has been provided.